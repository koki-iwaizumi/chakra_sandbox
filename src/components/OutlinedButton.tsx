import { FC } from "react";
import { Button, SpaceProps } from "@chakra-ui/react";

export interface Props {
  text: string;
  m?: SpaceProps["m"];
}

export const OutlinedButton: FC<Props> = (props: Props) => {
  return (
    <Button colorScheme="blue" m={props.m}>{`${props.text}:${props.m}`}</Button>
  );
};
