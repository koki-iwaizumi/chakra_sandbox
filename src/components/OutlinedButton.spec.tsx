import "@testing-library/jest-dom";
import { render } from "@testing-library/react";
import { OutlinedButton } from "./OutlinedButton";

describe("OutlinedButton", () => {
  it("should foo", () => {
    const { asFragment } = render(<OutlinedButton text="save" m={1} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
