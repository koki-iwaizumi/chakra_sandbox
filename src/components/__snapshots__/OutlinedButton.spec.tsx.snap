// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OutlinedButton should foo 1`] = `
<DocumentFragment>
  <button
    class="chakra-button css-i7psh6"
    type="button"
  >
    save:1
  </button>
</DocumentFragment>
`;
